html/pages/index.html

```diff
@@ -3,180 +3,269 @@
 title: Movies&amp;Stuff
 ---
 
-<nav class="navigation">
-  <h1 class="logo"><a href="#">Movies&amp;Stuff</a></h1>
-
-  <ul class="navigation-menu">
-    <li class="navigation-menu-item">
-      <a href="#movies">Movies</a>
-    </li>
-    <li class="navigation-menu-item">
-      <a href="#tv-shows">TV Shows</a>
-    </li>
-    <li class="navigation-menu-item navigation-button">
-      <a href="#subscribe">Subscribe</a>
-    </li>
-  </ul>
+<nav class=" nav navbar navbar-light">
+  <div class="container">
+    <button class="navbar-toggler hidden-md-up float-xs-right"
+            type="button"
+            data-toggle="collapse"
+            data-target="#navbarResponsive"
+            aria-controls="navbarResponsive"
+            aria-expanded="false"
+            aria-label="Toggle navigation">
```

Comment:
The mobile menu works fine, but we need a logo to be displayed even on mobile devices. Now it is collapsed. Please, investigate this issue.